<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RayTracing | Pre-alpha 0.5v</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_theory">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 20px;
border: 2px solid #094065;</string>
    </property>
    <property name="text">
     <string>Теория</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_demo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 20px;
border: 2px solid #094065;</string>
    </property>
    <property name="text">
     <string>Демонстрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_test">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 20px;
border: 2px solid #094065;</string>
    </property>
    <property name="text">
     <string>Тестирование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Учебно-демонстрационная программа обучения основам рендеринга</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Профиль</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выход из профиля</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Контекстная справка</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
